tests/pytests/test_multithread.py

```diff
@@ -152,7 +152,7 @@ def test_burst_threads_sanity():
     expected_total_jobs = 0
     for algo in VECSIM_ALGOS:
         additional_params = ['EF_CONSTRUCTION', n_vectors, 'EF_RUNTIME', n_vectors] if algo == 'HNSW' else []
-        for data_type in VECSIM_DATA_TYPES:
+        for data_type in VECSIM_DATA_TYPES_EXTENDED:
```

Comment:
Why testing the new types only here? Consider testing in test that us using `mt_mode_full` as well